### PR TITLE
C1s13 know how prizes

### PR DESCRIPTION
**Pull request by adamggg**
Scenario Reviewer: @KareemAliRahman
Implementation Reviewer: @NourhanMohamed
UML Reviewer: @Khloud-Khalid
Documentation Reviewer: @zoheiry

### CODE REVIEW

**Comment by adamggg** [comment]
[#174]

File: arability/app/views/games/_show_prizes.html.erb [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<div class="award-zone" id="award-list">
+  <div class="won">
+    <% if I18n.locale == :en %>
+      <a class="pull-right btn btn-success" style="padding: 5px; margin-right: 12px; margin-top: 28px;" onclick="backToGame()">Back</a>
+    <% else %>
+      <a class="pull-left btn btn-success" style="padding: 5px; margin-left: 28px; margin-top: 28px;" onclick="backToGame()">رجوع </a>
+    <% end %>
+    <h3 class="title-head"><%= t(:won_prizes_title) %></h3>
+    <% if @won_prizes.size == 0 %>
```
Comment (NourhanMohamed) [inline arability/app/views/games/_show_prizes.html.erb]: better use .blank?

File: arability/app/views/games/_show_prizes.html.erb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            <%= t(:score) %> <%= prize.score %>
+          </div>
+        </div>
+      <% end %>
+    <% end %>
+  </div>
+
+  <div class="not-won">
+    <h3 class="title-head"><%= t(:not_won_prizes_title) %></h3>
+    <% if @not_won_prizes.size == 0 %>
```
Comment (NourhanMohamed) [inline arability/app/views/games/_show_prizes.html.erb]: better use .blank?

File: arability/spec/models/show_prizes_spec.rb [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+    prize2
+    prize3
+    prize4
+    prize5
+    prize6
+    prize7
+  end
+
+  it "adam doesn't have any prizes" do
+    gamer_adam.id.to_i.should_not eq(nil)
```
Comment (NourhanMohamed) [inline arability/spec/models/show_prizes_spec.rb:110]: you could use be_nil instead of eq(nil)
But that's just a side note not a review comment =)

**Comment by krmali** [comment]
@adamggg , solve conflicts

**Pull request merged**